conda/install.py

```diff
@@ -56,7 +56,7 @@ def __enter__(self):
         def __exit__(self, exc_type, exc_value, traceback):
             pass
 
-on_win = bool(sys.platform == 'win32')
+on_win = bool(sys.platform.startswith("win"))
 
```

Comment:
IMO the safe variant is checking for `win32` because if that is ever changed, probably a few assumptions are also changed and this might result in code you wanted to guard with this. (ok, this is probably a really unrealistic scenario given some stories about MS not abandoning any windows API even if it is 30 years old and actually a bug...)